<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>287</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>움직임 감지 알람</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>STATUS</string>
    </property>
    <widget class="QLabel" name="status_alert_d">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>unknown</string>
     </property>
    </widget>
    <widget class="QLabel" name="status_es_d">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>unknown</string>
     </property>
    </widget>
    <widget class="QLabel" name="status_alert">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Alert</string>
     </property>
    </widget>
    <widget class="QLabel" name="status_es">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ES Server</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_alert">
    <property name="title">
     <string>Alert</string>
    </property>
    <addaction name="alert_enable"/>
    <addaction name="alert_disable"/>
   </widget>
   <widget class="QMenu" name="menu_option">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="menu_ES_Server"/>
   </widget>
   <widget class="QMenu" name="menu_info">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionhelp"/>
    <addaction name="separator"/>
    <addaction name="actionInfo"/>
   </widget>
   <addaction name="menu_alert"/>
   <addaction name="menu_option"/>
   <addaction name="menu_info"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="alert_enable">
   <property name="text">
    <string>Enable</string>
   </property>
   <property name="iconText">
    <string>Enable</string>
   </property>
  </action>
  <action name="alert_disable">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disable</string>
   </property>
  </action>
  <action name="menu_ES_Server">
   <property name="text">
    <string>ES Server</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
